client/gutenberg/extensions/vr/save.jsx

```diff
@@ -0,0 +1,23 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default ( { attributes, className } ) => (
+	<div className={ className }>
+		<iframe
+			title={ __( 'VR Image', 'jetpack' ) }
+			allowFullScreen="true"
+			frameBorder="0"
+			width="100%"
+			height="300"
+			src={
+				'https://vr.me.sh/view/?view=' +
```

Comment:
Gutenberg has a nice `addQueryArgs` utility we can use for building the query string - https://github.com/WordPress/gutenberg/blob/adb1f0e6e8271ce2b9c49963444add331ae90017/packages/url/src/index.js#L29

Reply:
#27929